### PR TITLE
Ports gibbers to Initialize

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by Iamgoofball** [comment]
real talk does anything actually use the autogibber

**Comment by SgtxRootbeer** [comment]
Auto gibber?

**Comment by Iamgoofball** [comment]
exactly my point

**Comment by SgtxRootbeer** [comment]
@iamgoofball
I don't know what it is, but it sounds like something useful for cargo. Can you please explain the autogibber to me?

**Comment by Iamgoofball** [comment]
see: the title

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm [hunk 1]
```diff
@@ -45,10 +44,10 @@
 			M.gib()
 
 
-/obj/machinery/gibber/New()
-	..()
+/obj/machinery/gibber/Initialize()
+	. = ..()
 	add_overlay("grjam")
-	var/obj/item/weapon/circuitboard/machine/B = new /obj/item/weapon/circuitboard/machine/gibber(null)
+	var/obj/item/weapon/circuitboard/machine/gibber/B = new
```
Comment (coiax) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm]: Wait, machine circuitboards are in null space? Color me unhappy.

Comment (lzimann) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm]: I believe they go inside with the next line where they are actually applied(I might be wrong though).

**Pull request merged**